calyx-py/test/correctness/queues/binheap.py

```diff
@@ -29,107 +31,369 @@ def insert_swap(prog, name, width, len, idx_w):
     return comp
 
 
-def insert_binheap(prog, name):
+def insert_binheap(prog, name, factor):
     """Inserts the component `binheap` into the program.
 
     It is a minimum binary heap, represented as an array.
 
     The heap just supports the `push` operation.
-    Its only input is `value`, the value to push to the queue.
+    Its only inputs are `value` and `rank`, the value and rank to push to the queue.
```

Comment:
A slightly more descriptive docstring may again help the next person. In particular, `value` will be pushed _with_ `rank`. 

Also quick catch, you actually have three inputs now that `cmd` has been added!